openapi/src/json-schemas/api/resources/metadata-resource.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Metadata resource",
+    "allOf": [
+        {"$ref": "../json-api-base-resource.json"},
+        {
+            "properties": {
+                "type": {
+                    "const": "metadata"
+                },
+                "attributes": {
+                    "allOf": [
+                        {
+                            "required": [
+                                "questionnaire-document-version",
+                                "questionnaire-id",
+                                "created",
+                                "modified",
+                                "submission-status",
+                                "expires"
+                            ],
+                            "properties": {
+                                "questionnaire-document-version": {
+                                    "enum": ["4.2.0"]
```

Comment:
After discussion, it has transpired that there is not requirement for this property at this time. Just remove it and it will reduce this complexity that we previously spoke about.